clients/go/gengo.go

```diff
@@ -281,79 +281,74 @@ func methodCode(s *spec.Swagger, op *spec.Operation, basePath, method, methodPat
 
 }
 
+func paramToTemplate(s *spec.Swagger, param *spec.Parameter, op *spec.Operation) paramTemplate {
```

Comment:
I think it would be ideal to have a comment explaining what this function does - I have no idea.